src/index.ejs

```diff
@@ -85,7 +85,11 @@
           onComplete: function() {
             /*callback for when */ console.log("everything is complete")
           },
-          steps: steps
+          steps: steps,
+          language: {
```

Comment:
is this object required? it shouldn't be.

Reply:
No, it's not required